extra/sponge.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
```

Comment:
Including this file could be useful. Is there an IntelliJ equivalent worth adding as well?